var/ramble/repos/builtin/modifiers/lscpu/modifier.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.modkit import *  # noqa: F403
+
+
+class Lscpu(BasicModifier):
+    """Define a modifier for lspcu
+
+    lscpu gives useful information about the underlying compute platform. This
+    modifier allows experiments to easily extract system information while the
+    experiment is being performed.
+    """
+    name = "lscpu"
+
+    tags = ['system-info', 'sysinfo', 'platform-info']
+
+    mode('standard', description='Standard execution mode for lscpu')
+
+    variable_modification('lscpu_log', '{experiment_run_dir}/lscpu_output.log', method='set', modes=['standard'])
+
+    archive_pattern('lscpu_output.log')
+
+    figure_of_merit_context('architecture', regex=r'Architecture:\s+(?P<arch>[\w-]+)', output_format='{arch}')
```

Comment:
Similar feedback here to my previous comment, I'm curious if we can do a better job of marking these as coming from a mod at analyze time, especially for uploaded or historic data

Reply:
We discussed this offline, and I added a task to fix this after variable expansion is supported in FOM names.